packages/pwa/app/components/_app/index.jsx

```diff
@@ -102,15 +116,18 @@ const App = (props) => {
 
     const onLogoClick = () => {
         // Goto the home page.
-        const path = homeUrlBuilder(HOME_HREF, targetLocale)
+        const path = homeUrlBuilder(HOME_HREF, {locale: targetLocale, site})
         history.push(path)
 
         // Close the drawer.
         onClose()
     }
 
     const onCartClick = () => {
-        const path = buildPathWithUrlConfig('/cart', {locale: targetLocale})
+        const path = buildPathWithUrlConfig('/cart', {
+            locale: targetLocale,
+            site: site.alias || site.id
+        })
```

Comment:
The url configuration we are passing in is defined inline 3 separate times for cart, account, and wishlist clicks.. is it possible to define the below once and use it in all 3 places? 

```
{
    locale: targetLocale,
    site: site.alias || site.id
}
```

Reply:
Yea, I've made them into a variable and reuse them in those functions